deployment/src/main/java/io/quarkiverse/quinoa/deployment/QuinoaProcessor.java

```diff
@@ -225,9 +227,13 @@ public void produceGeneratedStaticResources(
             return;
         }
         if (uiResources.isPresent() && !uiResources.get().resources().isEmpty()) {
+            String uiRootPath = QuinoaConfig.getNormalizedUiRootPath(configuredQuinoa.resolvedConfig());
             for (BuiltResourcesBuildItem.BuiltResource resource : uiResources.get().resources()) {
+                // note how uiRootPath always starts and ends in a slash
+                // and resource.name() always starts in a slash
```

Comment:
from the comments it means there are two slashes no? :)

Reply:
Should I change the comment to say `resource.name() always starts in a slash, therefore resource.name().substring(1) does not`?

Reply:
I changed the comment to `and resource.name() always starts in a slash, therfore resource.name().substring(1) never starts in a slash` in commit 083e3f4ce57f65920f66b390e36cf9d1498752fd